apps/shell/tash_command.c

```diff
@@ -561,12 +561,11 @@ int tash_execute_cmd(char **args, int argc)
 			if (tash_cmds_info.cmd[cmd_idx].exec_type == TASH_EXECMD_SYNC) {
 				/* function call to execute SYNC command */
 #ifdef CONFIG_PM
-				pm_arg_t pm_arg = {PM_IDLE_DOMAIN, PM_NORMAL};
 				pmdrv_fd = open("/dev/pm", O_WRONLY);
 				if (pmdrv_fd < 0) {
 					shdbg("open /dev/pm failed(%d), \n", get_errno());
 				} else {
-					pm_suspended = ioctl(pmdrv_fd, PMIOC_SUSPEND, &pm_arg);
```

Comment:
Hi @ewoodev @gSahitya-samsung @ritesh55555 , the argument for suspend/resume now takes domain instead of PM state. Please revise this part accordingly. Thank you.